mdc_weight_mgmt_xml_import/models/maintenance_equipment.py

```diff
@@ -0,0 +1,156 @@
+# # © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# # License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models, api, _, fields
+from datetime import datetime
+from lxml import etree as ET
+from pathlib import Path
+import traceback
+import logging
+import re
+
+
+class MaintenanceEquipment(models.Model):
+    _inherit = 'maintenance.equipment'
+
+    mdc_xml_import_active = fields.Boolean(
+        string="XML File Import Active"
+    )
```

Comment:
Set explicit default value in this case:
```suggestion
    mdc_xml_import_active = fields.Boolean(
        string="XML File Import Active",
        default=False,
    )
```